thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -3454,7 +3454,9 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                 // NB: when preserveGroups, the lhs group will always be complete at same time, so this will traverse whole group
                 if (!joinGroup->hasFlag(GroupFlags::gf_head))
                     return; // intermediate rows are completing, but can't output any of those until head finishes, at which point head marker will shift to next if necessary (see below)
+                #ifdef _DEBUG
```

Comment:
trivial/style: we normally don't indent #if*'s like this

Reply:
Pushed a squashed update that only changed the indentation.